src/style.css

```diff
@@ -0,0 +1,123 @@
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.search-bar {
+  position: relative;
+  margin-top: 20px;
+  display: flex;
+  width: 100%;
+}
+
+.search-imput-big {
+  height: 70px;
+  width: 100%;
+  background: white;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 62px;
+  line-height: 22px;
+  background-size: 19px;
+  background-position: 26px 25px;
+  font-size: 16px;
+  font-style: inherit;
+}
+
+.search-imput-small {
+  height: 42px;
+  width: 100%;
+  background: white;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 32px;
+  line-height: 11px;
+  background-size: 11px;
+  background-position: 13px 15px;
+  font-size: 14px;
+  font-style: inherit;
+}
+
+.search-imput-big:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  background-size: 19px;
+  background-position: 26px 25px;
+}
+
+.search-imput-small:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 32px;
+  line-height: 11px;
+  background-size: 11px;
+  background-position: 13px 15px;
+}
+
+.search-imput-small:focus {
+  background: linear-gradient(180deg, white 0%, #f6f6f7 100%);
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 32px;
+  line-height: 11px;
+  background-size: 11px;
+  background-position: 13px 15px;
+  outline: none;
+
+}
+
+.search-imput-big:focus {
+  background: linear-gradient(180deg, white 0%, #f6f6f7 100%);
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  background-size: 19px;
+  background-position: 26px 25px;
+  outline: none;
+}
+
+.search-imput-big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-imput-small::placeholder {
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
```

Comment:
make common class for both inputs and additional class for small inputs.
check same cases for other pairs of elements
![Screenshot from 2023-03-25 18-43-35](https://user-images.githubusercontent.com/33029864/227730607-236408f9-135a-484f-8d45-498ad257967f.png)